src/scripts/main.js

```diff
@@ -1 +1,12 @@
 'use strict';
+
+const thumbsList = document.getElementById('thumbs');
+const largeImg = document.getElementById('largeImg');
+
+thumbsList.addEventListener('click', (clickEvent) => {
+  const imageUrl = clickEvent.target.closest('a').href;
+
+  largeImg.src = imageUrl;
+
+  clickEvent.preventDefault();
```

Comment:
There is no sense to prevent the default action at the end of the method. Move it to the beginning of the function to avoid different unpredictable side effects